### ISSUE TITLE
Package.activateConfig is deprecated.

### ISSUE THREAD

**Issue by JamieREvans** [v3ss0n/goto#38]
Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details

```
Package.activateConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:263:11)
Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:234:14)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:772:25)
Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:219:20)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
```

**Comment by magalhas** [comment]
Actually I'm still facing this issue.

**Comment by fredcamps** [comment]
@JamieREvans @magalhas me 2

**Comment by manoj-apare** [comment]
Me 2 (in Atom 1.0). Deprecation Cop shows the following message:

Use a config schema instead. See the configuration section of https://atom.io/docs/latest/hacking-atom-package-word-count and https://atom.io/docs/api/latest/Config for more details

**Comment by sanstream** [comment]
Me 3.

**Comment by v3ss0n** [comment]
ok i will look into it.

**Comment by acusti** [comment]
Should be fixed in 1.8.0
